### ISSUE TITLE
Add links to wiki pages in the tasks

### ISSUE THREAD

**Issue by javiersanp** [Zverik/osmstreak#26]
I'm traslating the project and I think that more links to the Project Wiki would be absolutelly necessary for better training of new contributors. Each time you mention a tag there should be a link to the corresponding page. There should be also a link in terms like Addresses, Buildings, Turn restriction, Subways, etc.

Doing this, you will also take into account existing alternative mapping schemas. For example, in the sidewalks task, they can be draw as a separated way or with tags in the road.

**Comment by tohaklim** [comment]
Good idea, maybe as a hyperlink in the task description. The tree tasks already has wiki links

**Comment by javiersanp** [comment]
Maybe you can use a markdown translator?

**Comment by govvin** [comment]
I like this suggestion. A short,  gentle summary of how it's done so the new user isn't overwhelmed by the information from the page could be nice.
